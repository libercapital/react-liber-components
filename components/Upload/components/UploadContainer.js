"use strict";

require("core-js/modules/es6.object.define-property");

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _react = _interopRequireDefault(require("react"));

var _reactRedux = require("react-redux");

var _Upload = _interopRequireDefault(require("./Upload"));

var _store = _interopRequireDefault(require("../store"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var _default = function _default(props) {
  return _react["default"].createElement(_reactRedux.Provider, {
    store: _store["default"]
  }, _react["default"].createElement(_Upload["default"], props));
};

exports["default"] = _default;