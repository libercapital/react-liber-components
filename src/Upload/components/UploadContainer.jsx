import React from 'react';
import { Provider } from 'react-redux';
import Upload from './Upload';
import store from '../store';

export default props => (
  <Provider store={store}>
    <Upload {...props} />
  </Provider>
);
