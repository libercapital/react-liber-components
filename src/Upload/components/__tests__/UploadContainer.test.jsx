import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import UploadContainer from '../UploadContainer';

describe('Testing UploadContainer Component', () => {
  let props;
  configure({ adapter: new Adapter() });
  beforeEach(() => {
    props = {
      files: {},
    };
  });
  it('should render correctly ', () => {
    const wrapper = shallow(<UploadContainer {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
