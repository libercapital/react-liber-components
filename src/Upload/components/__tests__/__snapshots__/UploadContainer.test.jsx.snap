// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing UploadContainer Component should render correctly  1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(Component)
    files={Object {}}
  />
</Provider>
`;
